RadixWallet/Features/AccountPreferencesFeature/Children/ThirdPartyDeposits/ResourcesList+Reducer.swift

```diff
@@ -194,9 +194,13 @@ public struct ResourcesList: FeatureReducer, Sendable {
 			state.loadedResources.removeAll(where: { $0.address == resource })
 			switch resource {
 			case let .assetException(resource):
-				state.thirdPartyDeposits.assetsExceptionList.removeAll(where: { $0.address == resource.address })
+				state.thirdPartyDeposits.updateAssetsExceptionList {
```

Comment:
I guess these should end in -set now?

Reply:
perhaps, but the stored property is still called List.